_articles/introduction-to-panorama-ui-with-typescript.md

```diff
@@ -25,37 +25,30 @@ The name Typescript comes from the fact that the language is basically Javascrip
 *   Remember to compile
 *   Requires good definitions for Panorama
 
-## How to install Typescript
+## How to install TypeScript
 
-Step 1: Install [Node.js](https://nodejs.org/en/) which is used to compile Typescript.
+1. Install [Node.js](https://nodejs.org/en/) which is used to compile TypeScript.
+2. Create a `package.json` file in the root directory of your project with at least `{}` content.
+3. Install required dependencies by opening a command prompt and executing `npm install -D typescript panorama-types`.
```

Comment:
Why not `npm install --save-dev typescript panorama-types`, allowing you to remove step 2?

Reply:
It doesn't create `package.json` file when it is missing, it just installs it without saving. We can replace that step with ``run `npm init` ``, but it creates a lot of unnecessary (for an application) fields.